src/executables/uber.js

```diff
@@ -6,5 +6,5 @@ import pkg from '../../package.json';
 
 program.version(pkg.version)
   .command('price', 'get price estimate')
-  .command('time', 'get time to pickup estimate')
+  .command('time <address>', 'get time to pickup estimate')
```

Comment:
Yes, because that's part of the documentation for the CLI, from what I recall. One of the problems I had using the application was understanding where to put the parameters. Small niggle but I don't think it hurts.